Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -172,7 +172,7 @@ def bump_formula_pr
           EOS
         end
         check_new_version(formula, tap_remote_repo, url: old_url, tag: new_tag, args: args) if new_version.blank?
-        resource_path, forced_version = fetch_resource(formula, new_version, old_url, tag: new_tag)
+        resource_path, forced_version = fetch_resource_and_forced_version(formula, new_version, old_url, tag: new_tag)
```

Comment:
This collided with a method name in `Library/Homebrew/cmd/fetch.rb` in the same namespace, so i chose to rename the one in `dev-cmd`